xarray/core/resample_cftime.py

```diff
@@ -54,29 +54,72 @@
 )
 from ..coding.cftimeindex import CFTimeIndex
 
+try:
+    import cftime
```

Comment:
Should cftime be imported lazy? Not sure how much import time this adds.

Reply:
I went ahead and made this lazy.  There are a few other places where we follow this pattern as well.  Maybe I'll address that in a separate PR.